### DISCUSSION TITLE
On-Prem minio object name case sensitive

### DISCUSSION THREAD

**Discussion by mkiran1010** [minio/minio#20431] [Q&A] [answered]
I’m encountering an issue with MinIO where object names appear to be case-sensitive. This case sensitivity is affecting some of my operations. Has anyone else experienced this problem? If so, could you share any solutions or settings adjustments that might help?

![image](https://github.com/user-attachments/assets/8a2c3b7f-e022-4952-aebf-fcc304aa53a2)
but when i try access
[maindomain.com/media/tr_xxxxxxxxxxx.pdf](http://maindomain.com/media/tr_xxxxxxxxxxx.pdf)

<Error>

<Code>NoSuchKey</Code>
<Message>The specified key does not exist.</Message>
<Key>documents/pdf/tr_xxxxxx_sASoA91.pdf</Key>
<BucketName>django-media</BucketName>
<Resource>/django-media/documents/pdf/tr_xxxx_sASoA91.pdf</Resource>
<RequestId>xxxx</RequestId>
<HostId>xxxxxxx</HostId>

</Error>

**Comment by harshavardhana** [comment, marked as answer]
Object names are case sensitive - as per AWS S3 compliant behavior.
